### PR TITLE
#185528211 Redesign of the seller dashboard

### PR DESCRIPTION
**Pull request by frankflej**
## What does this PR do?
This PR aims at redesigning the dashboard of the seller
## Description of Task to be completed?
This task focuses on the implementation of the seller's dashboard design on both mobile and bigger screens(responsiveness)
## How should this be manually tested?

- Clone the repository using `git clone https://github.com/atlp-rwanda/e-comm-team-techsmith-fn`

- Checkout to the branch using `git checkout bg-dashboard-design-185528211`

- Install node packages that the application depends on: `npm i`

- Create a `.env` file in the root directory and add all environment variables required to run the application. The required variables are listed in the `.env.example` file available in root directory of the application.

- Start the application development server: `npm start`

- Check out the components documentation: `npm run storybook`

## What are the relevant pivotal tracker/Trello stories?
[#185528211](https://www.pivotaltracker.com/story/show/185528211)
## Screenshots (if appropriate)
Screen Shot 2015-11-07 at 1 07 03 PM
Screen Shot 2015-11-07 at 1 32 46 PM

### CODE REVIEW

**Review by mken1205 — APPROVED** [review]

File: src/containers/SellerProductsContainer.jsx
Comment (irakevine) [inline src/containers/SellerProductsContainer.jsx:1]: great that you removed this min-h-[80vh],
somehow it made the design look bad

Comment (irakevine) [inline src/containers/SellerProductsContainer.jsx:1]: with this: flex, h-[80vh] justify-evenly., that was also causing it to not work properly, you did well to fix it!

**Review by ReneNtabana — APPROVED** [review]

**Review by irakevine — APPROVED** [review]
Great Work!

**Review by niyontwali — APPROVED** [review]

**Review by bergad37 — COMMENTED** [review]
Great job done!!

**Pull request merged**